app/src/main/java/com/github/neighbortrader/foodboardapp/clientmodel/Offer.java

```diff
@@ -66,7 +68,7 @@ public static Offer createOfferFromJSON(JSONObject jsonObject) {
             LocalDateTime purchaseDate = LocalDateTime.parse(jsonObject.getString("purchaseDate"), dateTimeFormatter);
             LocalDateTime expireDate = LocalDateTime.parse(jsonObject.getString("expireDate"), dateTimeFormatter);
 
-            return new Offer(price, Grocery.findGrocery(grocerieId), description, purchaseDate, expireDate);
+            return new Offer(User.getCurrentUserInstance(), price, Grocery.findGrocery(grocerieId), description, purchaseDate, expireDate);
```

Comment:
du gibst ein Objekt im Konstruktor zurück ? seh ich das richtig ???

Reply:
nein

Reply:
sondern was wird hier returned und in welcher Methode bin ich ?

Reply:
Das funktioniert auch nicht in Java. Auf Kommentare mit mehr als einem Fragezeichen antwortet man eigentlich nicht :p